src/components/Card/Card.jsx

```diff
@@ -27,17 +28,37 @@ const { prefix, iotPrefix } = settings;
 const OptimizedSkeletonText = React.memo(SkeletonText);
 
 /** Full card */
-const CardWrapper = styled.div`
-  background: white;
-  height: ${props => props.dimensions.y}px;
-  ${props => (props.isExpanded ? 'height: 100%; width: 100%;' : '')}
-  display: flex;
-  flex-direction: column;
-  span#timeRange {
-    display: ${props => (props.cardWidthSize < 230 ? `none` : `flex`)};
-  }
-  overflow: ${props => (props.showOverflow ? `visible` : `hidden`)};
-`;
+const CardWrapper = ({
+  children,
+  dimensions,
+  isExpanded,
+  showOverflow,
+  id,
+  style,
+  className,
+  ...others
+}) => {
+  // TODO: Solve this better by either
+  // a) explicitly define and pass along the HTML attributes needed for the wrapper
```

Comment:
I'd prefer this route, but we can see what @davidicus and @scottdickerson think. We have similar issues in quite a few components where we somewhat blindly pass those `...other` props

Reply:
Yeah, it would be nice to have an explicit API defining what's required for positioning.

Reply:
Do we know of any reasons to pass on additional attributes for  the `Card` container `div`? Perhaps we just add a `data-testid` for the div and not spread any props at all. I like the flexibility of spreading props but it's definitely error prone when being applied to a DOM element.

Reply:
Now I'm explicitly adding the event handlers that are attached by the Dashboard grid when "isEditable" is active. **But, since I can't be sure that other applications aren't adding additional HTML attributes** (this was allowed by Styled Components) I've created a component utility filter method to filter all remaining props and only pass forward valid HTML attributes or react library props like 'ref' etc. Our own custom props that "accidentally" have been forwarded will be filtered out.

The source of my list of valid HTML attributes is the Styled Components source code. My idea behind this is that we can use this new `filterValidAttributes` when we refactor Styled Components and face a similar situations.

Also Adding the 'data-testid' to the card and the wrapper